qiita_db/study.py

```diff
@@ -1152,6 +1137,75 @@ def unshare(self, user):
             qdb.sql_connection.TRN.add(sql, [self._id, user.id])
             qdb.sql_connection.TRN.execute()
 
+    def add_tags(self, user, tags):
+        """Sets the tags of the study
+
+        Parameters
+        ----------
+        user: User object
+            The user to unshare the study with
+        tags : list of str
+            The tags to add to the study
```

Comment:
The tags to attach to the study - or something along those lines (since it is not only adding)